package.json

```diff
@@ -30,7 +30,7 @@
     "@changesets/cli": "^2.26.2",
     "@testing-library/jest-dom": "^6.4.0",
     "@testing-library/react": "^14.2.0",
-    "@types/jest": "^29.5.11",
+    "@types/jest": "^29.5.12",
```

Comment:
Dope,
also can you please run `yarn changeset` as patch and describe the change for this PR.